lib/td_core/search/cluster.ex

```diff
@@ -8,9 +8,10 @@ defmodule TdCore.Search.Cluster do
     indexes =
       Enum.reduce(aliases, %{}, fn
         {index, alias_name}, acc ->
-          with %{settings: settings} = config <- Map.fetch!(indexes, index),
-               %{} = config <- Map.put(config, :settings, Map.merge(settings, defaults)) do
-            Map.put(acc, String.to_atom(alias_name), config)
+          with config <- Keyword.fetch!(indexes, index),
+               settings <- Keyword.fetch!(config, :settings),
+               [_ | _] = config <- Keyword.put(config, :settings, Map.merge(settings, defaults)) do
+            Map.put(acc, String.to_atom(alias_name), Map.new(config))
```

Comment:
Use `String.to_existing_atom` instead of .to_atom to avoid dynamic atoms creation